projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.scss

```diff
@@ -2,10 +2,11 @@
 @import '../../../styles/mixins';
 
 .go-calendar {
-  position: absolute;
+  background: $theme-light-bg;
   border: 1px solid $theme-light-border;
   border-radius: $global-radius;
-  background: $theme-light-bg;
+  position: absolute;
+  right: 0;
```

Comment:
If we're always going to be showing this on the right should we remove the logic around `displayFromRight` that depending on where it's on the screen it displays it on the right?

Reply:
That's a good observation. I made some modifications around this logic. Would you mind taking another look?